R/followers.R

```diff
@@ -7,12 +7,8 @@ friends <- function(user_id, n=NULL, ...) {
 }
 
 ffBase <- function(type, user_id, n=NULL, ...) {
-  if (can_access_other_account(user_id)) {
-    params <- parseUsers(user_id)
-    doCursorAPICall(paste(type, 'ids', sep='/'), 'ids', num=n, params=params, method='GET', ...)
-  } else {
-    warning("Can not lookup relationships for user id ", user_id)
-  }
+  params <- parseUsers(user_id)
```

Comment:
So my concern about removing can_access_other_account is that the reason I added it in was that "why did I not receive the results I expected?" is a mega-super-FAQ and this provides a warning() for a pretty common case of why people didn't receive the results they expected.

Is there some way to get the behavior you're after w/o removing this check?